README.md

```diff
@@ -62,12 +62,12 @@ The coverage reports are written in each module's `target` directory and aggrega
 #### Menas requirements:
 - [**Tomcat 8.5/9.0** installation](https://tomcat.apache.org/download-90.cgi)
 - [**MongoDB 4.0** installation](https://docs.mongodb.com/manual/administration/install-community/)
-- [**Spline UI deployment**](https://absaoss.github.io/spline/) - place the [spline.war](https://search.maven.org/remotecontent?filepath=za/co/absa/spline/spline-web/0.3.9/spline-web-0.3.9.war)
- in your Tomcat webapps directory (rename after downloading to _spline.war_); NB! don't forget to set up the `spline.mongodb.url` configuration for the _war_
+- [**Spline service deployment**](https://absaoss.github.io/spline/#get-spline)
 - **HADOOP_CONF_DIR** environment variable, pointing to the location of your hadoop configuration (pointing to a hadoop installation)
 
-The _Spline UI_ can be omitted; in such case the **Menas** `za.co.absa.enceladus.spline.urlTemplate` setting should be set to empty string. 
-
+The _Spline service_ can be omitted; in such case the **Standardization** and **Conformance** `spline.producer.url` setting
+ so as **Menas** `lineageReadApiUrl` and `menas.oozie.lineageWriteApiUrl` settings should be all set to empty string. 
```

Comment:
I am not sure the `so as` makes sense to me here. `as well as`?

Reply:
`as well as` it is 👍